dev-python/hvac/hvac-0.2.17.ebuild

```diff
@@ -2,11 +2,11 @@
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
-PYTHON_COMPAT=( python{2_7,3_5,3_6} )
+PYTHON_COMPAT=( python3_{5,6} )
 
 inherit distutils-r1
 
-DESCRIPTION="Python 2/3 client for HashiCorp Vault"
+DESCRIPTION="Python 3 client for HashiCorp Vault"
```

Comment:
Is there a point in saying 'Python 3'? I could get if we're copying some upstream description but if we remove 'Python 2' from it, it makes no sense.